File: stripe/src/main/java/com/stripe/android/view/StripeIntentResultExtras.java
package com.stripe.android.view;

import com.stripe.android.StripeIntentResult;
import com.stripe.android.model.PaymentIntent;

public final class StripeIntentResultExtras {
    /**
     * Should be a {@link PaymentIntent#getClientSecret()}
     */
    public static final String CLIENT_SECRET = "client_secret";

    public static final String AUTH_EXCEPTION = "exception";

    /**
     * See {@link StripeIntentResult.Outcome} for possible values
     */
    public static final String FLOW_OUTCOME = "flow_outcome";

    private StripeIntentResultExtras() {
    }
}

File: stripe/src/main/java/com/stripe/android/view/StripeIntentResultExtras.kt
package com.stripe.android.view

import com.stripe.android.StripeIntentResult
import com.stripe.android.model.StripeIntent

internal object StripeIntentResultExtras {
    /**
     * Should be a [StripeIntent.clientSecret]
     */
    const val CLIENT_SECRET = "client_secret"

    const val AUTH_EXCEPTION = "exception"

    /**
     * See [StripeIntentResult.Outcome] for possible values
     */
    const val FLOW_OUTCOME = "flow_outcome"

    const val SHOULD_CANCEL_SOURCE = "should_cancel_source"

    const val SOURCE_ID = "source_id"

    const val SOURCE = "source"
}
